### PR TITLE
get_or_create tiers by only campaign id and tier id

### PR DESCRIPTION
**Pull request by evykassirer**
Previously, duplicate tiers were being created because `get_or_create` didn't get the exact tier (a mismatch on `tier_amount_cents` and created a second one. Now we `get_or_create` on fewer fields and add/update the rest.

### CODE REVIEW

File: Pipfile.lock [hunk 1]
```diff
@@ -18,10 +18,10 @@
     "default": {
         "certifi": {
             "hashes": [
-                "sha256:4c1d68a1408dd090d2f3a869aa94c3947cc1d967821d1ed303208c9f41f0f2f4",
```
Comment (evykassirer) [inline Pipfile.lock:21]: I don't know much about lock files and why these change like this and if that's something we'd want to commit

**Review by phildini — APPROVED** [review]
